src/styles/emojiFont/index.ios.js

```diff
@@ -0,0 +1,5 @@
+// Using system fonts e.g. Segoe UI Emoji results in error in ios
```

Comment:
Do we need to change the font for Android? If not, rename this file to `index.native.js`. Remove this comment.